ui/admin/app/components/form/host-set/aws/index.hbs

```diff
@@ -74,6 +74,94 @@
     </:body>
   </form.fieldset>
 
+  <Rose::Form::Fieldset>
+    <:title>{{t 'resources.host-set.form.preferred_endpoints.label'}}</:title>
+    <:description>{{t
+        'resources.host-set.form.preferred_endpoints.help'
+      }}</:description>
+    <:body>
+      {{#if @model.errors.preferred_endpoints}}
+        <Rose::Form::Errors as |errors|>
+          {{#each @model.errors.preferred_endpoints as |error|}}
+            <errors.message>{{error.message}}</errors.message>
+          {{/each}}
+        </Rose::Form::Errors>
+      {{/if}}
+      <Rose::Table @style='condensed' as |table|>
+        <table.header as |header|>
+          <header.row @hidden={{true}} as |row|>
+            <row.headerCell>{{t 'titles.value'}}</row.headerCell>
+            <row.headerCell />
+          </header.row>
+        </table.header>
+        <table.body as |body|>
+          {{#each @model.preferred_endpoints as |endpoint i|}}
+            <body.row as |row|>
+              <row.cell>
+                <form.input
+                  @type='text'
+                  @value={{endpoint.value}}
+                  @error={{@model.errors.preferred_endpoints}}
+                  @contextual={{true}}
+                  @placeholder='cidr:2001:4860:4860::8888/32'
```

Comment:
Does `placeholder` not work?  I would have thought a regular HTML attribute would suffice.

Reply:
oops it does work! I tried that initially and had some typo and went with the input attr route. I can update it now